plant-&amp;-cell-physiology.csl

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style class="in-text" version="1.0" et-al-min="6" et-al-use-first="6" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+<style class="in-text" version="1.0" et-al-min="7" et-al-use-first="6" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
   <info>
     <title>Plant and Cell Physiology</title>
-    <id>http://www.zotero.org/styles/plant-and-cell-physiology</id>
-    <link href="http://www.zotero.org/styles/plant-and-cell-physiology" rel="self"/>
+    <id>http://www.zotero.org/styles/plant-&amp;-cell-physiology</id>
+    <link href="http://www.zotero.org/styles/plant-&amp;-cell-physiology" rel="self"/>
```

Comment:
@damnation333, no, that's not right. We want to keep "plant-and-cell-physiology" for the style ID and file name. It's just the title that had to be changed: `<title>Plant &amp; Cell Physiology</title>`